test/commands/request/runRequest/sendArr.sas

```diff
@@ -0,0 +1,11 @@
+parmcards4;
```

Comment:
this statement should be removed, I'm surprised the test is passing with it in.  Same for below